library/component/badge/badge.css

```diff
@@ -0,0 +1,64 @@
+.badge{
+    position:relative;
+    font-size:3.5rem;
+    width:5rem;
+    height:5rem;
+  }
+  .left_badge_text{
+      position:absolute; 
+      background-color:var(--warning-color);
+      color:white;
+      font-size:1.5rem;
+      width:2rem;
+      height:2rem;
+      border-radius:50%;
+      text-align:center;
+      top:0;
+      left:0;
+      z-index:2;
+  }
+  .right_badge_text{
+    position:absolute; 
+    background-color:var(--danger-color);
+    color:white;
+    font-size:1.5rem;
+    width:2rem;
+    height:2rem;
+    border-radius:50%;
+    text-align:center;
+    top:0;
+    right:0;
+    z-index:2;
+  }
+  .badge_avatar{
+    width:7rem;
+    height:7rem;
+  }
+  .avatar_image{
+    width:100%;
+    height:100%;
+    border:none;
+    border-radius:50%;
+  }
+  .badge_avatar_successdot{
+    width:1rem;
+    height:1rem;
+    background-color:var(--success-color);
+    position:absolute;
+    bottom:0%;
+    right:15%;
+    border:1px solid var(--light);
```

Comment:
var(--light) make less sense maybe add colour at the start or end to show that it's a colour.

Reply:
ok I will do it , thanks zoef.